app/components/gh-nav-menu.js

```diff
@@ -11,13 +11,15 @@ export default Component.extend({
 
     open: false,
 
-    navMenuIcon: computed('config.blogUrl', function () {
-        let url = `${this.get('config.blogUrl')}/favicon.png`;
+    navMenuIcon: computed('config.blogUrl', 'settings.icon', function () {
+        let blogIcon = this.get('settings.icon') ? this.get('settings.icon') : 'favicon.ico';
```

Comment:
Ignore me, blank strings are falsey in JS. Clearly not had enough caffeine this afternoon 😬